### PR TITLE
Various small documentation related changes

### PR DESCRIPTION
**Pull request by hhorak**
Use maintainer label consistently
Few edits in top-level README
Remove tables in README, because go-md2man cannot convert them properly
Remove hard-coded /opt/app-root in Dockerfile

### CODE REVIEW

File: 2.4/root/usr/share/container-scripts/httpd/README.md [hunk 1]
```diff
@@ -59,21 +59,30 @@ $ docker run -d --name httpd -p 8080:8080 httpd-app
 
 The structure of httpd-app can look like this:
 
-| Folder name       | Description                |
-|-------------------|----------------------------|
-| `./httpd-cfg`     | Can contain additional Apache configuration files (`*.conf`)|
-| `./httpd-pre-init`| Can contain shell scripts (`*.sh`) that are sourced before `httpd` is started|
-| `./httpd-ssl`     | Can contain own SSL certificate (in `certs/` subdirectory) and key (in `private/` subdirectory)|
-| `./`              | Application source code |
+**`Folder name`**
```
Comment (pkubatrh) [inline 2.4/root/usr/share/container-scripts/httpd/README.md]: Is the original header supposed to be in here?

Comment (hhorak) [reply 2.4/root/usr/share/container-scripts/httpd/README.md]: It shouldn't, thanks for spotting this.

Comment (hhorak) [reply 2.4/root/usr/share/container-scripts/httpd/README.md]: Fixed now.

**Comment by pkubatrh** [comment]
lgtm now, thanks! Merging

**Pull request merged**